frontend/lib/tests/shop/order-items-snapshot-immutable.test.ts

```
@@ -216,15 +285,16 @@ describe('P0-6 snapshots: order_items immutability', () => {
       try {
-        await cleanupByIds({ orderId, productId });
+        await cleanupByIds({ orderId, productId, priceId });
       } catch (e) {
         cleanupError = e;
         console.error('[test cleanup failed]', { orderId, productId }, e);
       }
-    }
-    if (!primaryError && cleanupError) {
-      throw cleanupError;
+      if (!primaryError && cleanupError) {
+        throw cleanupError;
```

Biome (2.4.7)

[error] Unsafe usage of 'throw'.

(lint/correctness/noUnsafeFinally)